apollo-ios/Sources/ApolloTestSupport/SelectionSet+DictionaryIntializer.swift

```diff
@@ -0,0 +1,38 @@
+#if !COCOAPODS
```

Comment:
I put the new initialization method in ApolloTestSupport target to make sure this method is only used for mock purpose